docs/end-to-end-demo-rpi4.md

```diff
@@ -34,109 +43,147 @@ This demo will demonstrate how to get Akri working on a **Raspberry Pi 4**, all
     kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
     ```
 
-
 ## Set up mock udev video devices
 1. Open a new terminal and ssh into your ubuntu server that your cluster is running on.
-1. Install a kernel module to make v4l2 loopback video devices. Learn more about this module [here](https://github.com/umlaeute/v4l2loopback).
+1. To setup fake usb video devices, install the v4l2loopback kernel module and its prerequisites. Learn more about v4l2 loopback [here](https://github.com/umlaeute/v4l2loopback)
```

Comment:
Does this need these extra steps from the normal [e2e demo](https://github.com/deislabs/akri/blob/main/docs/end-to-end-demo.md#setting-up-mock-udev-video-devices):
```sh
sudo apt update
sudo apt -y install linux-modules-extra-$(uname -r)
sudo apt -y install dkms
```